src/components/Search.js

```diff
@@ -106,12 +114,12 @@ export function Search(props) {
                 </div>
 
                 <div className="uni-BodyHeader-extraOptions">
-                  {config.sorts?.length > 0 && (
+                  {sorts?.length > 1 && (
```

Comment:
We shouldn't need `?` anymore:

```suggestion
                  {sorts.length > 1 && (
```